doc/using_statistics.md

```diff
@@ -134,22 +136,93 @@ Server: Jetty(9.1.5.v20140505)
 }
 ```
 
+**Connecting The Focus To The Publisher For PubSub Method**
+
+Upon start of the Jitsi Videobridge, it will send the following stanza to the focus
```

Comment:
Thanks, trying to save others the time I've spent digging for answers. What do you mean by component here? In our setup, the focus scales our Videobridge cluster by monitoring the pubsub stats messages so we want the Videobridge to publish to the focus.